test/directory_watcher/linux_test.dart

```diff
@@ -17,8 +17,7 @@ void main() {
   sharedTests();
 
   test('DirectoryWatcher creates a LinuxDirectoryWatcher on Linux', () {
-    expect(
-        new DirectoryWatcher('.'), new isInstanceOf<LinuxDirectoryWatcher>());
+    expect(new DirectoryWatcher('.'), TypeMatcher<LinuxDirectoryWatcher>());
```

Comment:
If you're going to remove one `new`, should probably remove the other too. Or just keep the `new` for now and we'll remove them all later.

Reply:
I'll add back the `new`. We can run `dartfmt --fix` later (so as to keep this PR simple).